tests/test_config.py

```diff
@@ -24,6 +24,9 @@ def test_enabled_star():
 
             [[python]]
             python_path = /some/path
+
+            [[xpidl]]
+            header_bucket = /somewhere
```

Comment:
Is "bucket" a term commonly used with XPIDLs, or should we change this to "path" for consistency with other plugins (like "python_path")?